src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -256,10 +257,10 @@ import 'emby-ratingbutton';
 
         return new Promise(function (resolve, reject) {
             Promise.all([
-                import('appFooter-shared'),
-                import('itemShortcuts'),
-                import('css!./nowPlayingBar.css'),
-                import('emby-slider')
+                import('../appFooter/appFooter'),
+                import('../shortcuts'),
+                import('./nowPlayingBar.css'),
+                import('../../elements/emby-slider/emby-slider')
             ])
             .then(([appfooter, itemShortcuts]) => {
```

Comment:
```suggestion
            .then(([{default: appfooter}, itemShortcuts]) => {
```
or maybe there is an easier way

Reply:
i added all those modules at the beginning of the nowPlayerBar.js file. Since the css is independent from the rest, i don't see any reason to only load it when the player is needed (once loaded, it will remain always loaded). So the method getNowPlayingBar pass to be synchronous, and returns the now playing bar element.